db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,40 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
```

Comment:
@durandom 
* This challenge actually needs to get default to true since almost all auth types need this variable set to true, ill change it.
* Those booleans aren't related to the auth type, those are variables that are connected with authentication method on openshift-ansible.